yuglu-frontend/src/components/sign-up/sign-up.jsx

```diff
@@ -6,23 +6,36 @@ export default class SignUpPage extends React.Component {
     }
 
     handleInputChange(event) {
-        const name = event.target.name;
+        const target = event.target;
+        const name = target.name;
         this.setState({
             [name]: event.target.value
         });
     }
 
     handleSubmit(event) {
         event.preventDefault();
+        console.log(this.state)
+    fetch("http://localhost:3001/signup", {
+        method: 'POST',
+        cache: 'no-cache', 
+        credentials: 'same-origin', 
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        redirect: 'follow', 
+        referrer: 'no-referrer', 
+        body: JSON.stringify(this.state), 
+    })
+    .then(response => response.json()); 
```

Comment:
General feedback- should something happen when you have sent the request? Maybe it should redirect to a different page or do something else to show the form submitted successfully?